docs/monitoring/metrics.md

```diff
@@ -1117,7 +1117,7 @@ Metrics related to data exchange between task executors using netty network comm
       <td>Gauge</td>
     </tr>
     <tr>
-      <th rowspan="8">Task</th>
+      <th rowspan="9">Task</th>
```

Comment:
nit: side issue, can you merge both `Task` cells together (separate commit)? They are currently duplicated https://ci.apache.org/projects/flink/flink-docs-release-1.9/monitoring/metrics.html#default-shuffle-service
probably it's enough to change `rowspan=8` here to `16`, and remove the duplicated `<th rowspan="8"><strong>Task</strong></th>` from around L1167

Reply:
Ok.